Jropp/jason-ropp-js-iobp
# Database-Mini: Post/Put Users

## What It Does

This adds the saveNew and saveEdit functionality 

These two actions implement the sendRequest function. 

## How To Test

Save a user or edit a user and save it. The user list should update appropriately.

## Notes/Caveats

This does not fix sort functionality.

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:


## Dependencies

None
